.github/workflows/build-code-and-test.yml

```diff
@@ -0,0 +1,140 @@
+name: Build and test latest code
+
+on:
+  push:
+    paths-ignore:
+      - '**package.json'
+      - '**package-lock.json'
+      - '**Dockerfile'
```

Comment:
Is there a good reason to ignore this **Dockerfile

Reply:
If the Dockerfile changes then it should build the other workflow (core build).

We split it between core and code. If you change dependencies and run the code build you get unexpected results and it could be non-intuitive for developers.